content/concepts/asset-manager.md

```diff
@@ -203,7 +213,7 @@ Not comfortable with regular expressions? You can try yours [right here](https:/
 
 Let's take an example to make this clearer! 
 ```regexp
-/(?P<product_ref>.*)-.*-(?P<attribute_ref>.*)\.jpg/
+/(?P<product_ref>.*)\\_(?P<attribute_ref>.*)\\.jpg/
```

Comment:
```suggestion
/^(?P<product_ref>.*)_(?P<attribute_ref>.*)\\.jpg$/
```

Reply:
- no need to escape an underscore in a regexp
- I added the "beginning of string" (`^`) and "end of string" (`$`) anchors, which I believe will match most of the filenames